src/types.d.ts

```diff
@@ -47,12 +47,23 @@ export type BudgetApiConnection = {
   createBudget: (data: UnpersistedBudget) => Promise<Budget>
 }
 
+export type AccountComputedData = {
+  id: string
+  balance: string
+  reconciledBalance: string
+}
```

Comment:
I'm not sure if I like exporting this as its own type. I think it would be easier to understand what this does if we define it within `Account` and then use it as `Account.computedData`.